Userland/DevTools/HackStudio/LanguageServers/Cpp/ParserAutoComplete.h

```diff
@@ -59,6 +59,9 @@ class ParserAutoComplete : public AutoCompleteEngine {
         String m_text;
         OwnPtr<Preprocessor> m_preprocessor;
         OwnPtr<Parser> m_parser;
+
+        // FIXME: This HashTable must be re-computed if a declaration from a header file is modified
+        HashTable<RefPtr<Declaration>> m_declarations_from_headers;
```

Comment:
Does this ever need to store a null value?
Can we make it a `HashTable<NonnullRefPtr<Declaration>>`?